hazelcast/src/test/java/com/hazelcast/collection/impl/list/ListAbstractTest.java

```diff
@@ -535,6 +542,70 @@ public void testIteratorWithIndex() {
         }
     }
 
+    @Test
+    public void testCreateUseDestroyStress_doesNotThrow() throws InterruptedException {
```

Comment:
Should this test fail without the fix here? I didn't manage to see failure on master.

Reply:
I could not get it to fail either. It is probably because internal ArrayList does not run into a NullPointerException when it is concurrently expanded and destroyed. Internal hashmap in SetContainer on the other hand runs into NullPointerException pretty easily when it is modified concurrently. Nevertheless, I think it is nice to have this test for list also.

Reply:
makes sense